CONTRIBUTING.md

```diff
@@ -24,12 +25,16 @@ We're always looking to make Akita better, so definitely drop us a Feature Reque
 if you have any ideas brewing!
 
 Issue Templates you can choose from:
+
 - 📄 [Create New Blank Issue](https://github.com/esse-dev/akita/issues/new?assignees=&labels=&template=blank-issue.md&title=)
 - 🐞 [Create New Bug Report](https://github.com/esse-dev/akita/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D)
 - 💡 [Create New Feature Request](https://github.com/esse-dev/akita/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFEATURE+REQUEST%5D)
 
 ## Contribute Code
-	 
+
+Before contributing code to Akita please read [Akita's Privacy Policy](docs/PrivacyPolicy.md) so you
+can help us ensure data privacy for Akita users! If you make any changes that affect user data,
+please update the Privacy Policy as part of your changes.
```

Comment:
> please update the Privacy Policy as part of your changes.

suggested reword: `please review the Privacy Policy to check if any updates should be made to it alongside your changes`

and a nit: newline between this text and `### Implement Your Changes`